src/main/java/com/employee/controller/EmployeeController.java

```diff
@@ -181,6 +196,76 @@ public void findEmployeeByLastName(String lastName) {
 
 	}
 
+	public void findEmployeeByFirstName(String firstName) {
+		List<Employee> result;
+		try {
+			result = employeeService.getEmployeesByFirstName(firstName);
+
+			if (result != null) {
+
+				for (Employee employee : result) {
+					System.out.print(employee.getEmployeeId() + ", ");
+					System.out.print(employee.getFirstName() + " ");
+					System.out.print(employee.getLastName() + ", ");
+					System.out.print(employee.getEmail() + ", ");
+					System.out.print(employee.getHireDate() + ", ");
+					System.out.println();
+				}
+			} else {
+				System.out.println("User not found!");
+			}
+
+		} catch (SQLException e) {
+			e.printStackTrace();
+		}
+
+	}
+
+	public void getEmployeeByHireDate(String orderBy) {
+
+		if (orderBy == "ASC") {
+			getAllEmployeeByAscHireDate();
```

Comment:
@hninpwintaung-hpa You don't need to separate 2 methods for sorting.
Use only one method and sort based on user input. If ASC, use asc query, otherwise, use desc query. One method is enough.